README.md

```diff
@@ -115,13 +116,13 @@ Users need to have a basic understanding of the [Ansible concepts](https://docs.
   $ pip install --user ansible
```

Comment:
Can we remove `--user `from here .

Reply:
Currently we are thinking to add supported ansible version as 2.9.x

Reply:
Let me change this real quick and push another commit...

Reply:
Done